src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -66,11 +72,13 @@ public ModelManager(ReadOnlyUserProfile dukeCooks, ReadOnlyHealthRecords healthR
         this.userPrefs = new UserPrefs(userPrefs);
         this.workoutPlanner = new WorkoutPlanner(workoutPlanner);
         this.diaryRecords = new DiaryRecords(diary);
+        this.dashboards = new DashboardRecords(dashboards);
         filteredPersons = new FilteredList<>(this.userProfile.getUserProfileList());
         filteredRecords = new FilteredList<>(this.healthRecords.getHealthRecordsList());
         filteredRecipes = new FilteredList<>(this.recipeBook.getRecipeList());
         filteredExercises = new FilteredList<>(this.workoutPlanner.getExerciseList());
         filteredDiaries = new FilteredList<>(this.diaryRecords.getDiaryList());
+        filteredDashboards = new FilteredList<>(this.dashboards.getDashboardList());
```

Comment:
Similar to previous comment, should use the singular form if there is only one Dashboard